app/src/pages/inside/settingsPage/settingTabs/autoAnalysisTab/analysisForm/accuracyFormBlock/accuracyFormBlock.jsx

```diff
@@ -0,0 +1,169 @@
+  },
+  numberOfLogLinesAllOption: {
+    id: 'AccuracyFormBlock.numberOfLogLinesAllOption',
+    defaultMessage: 'All',
+  },
+  submitButtonText: {
+    id: 'AccuracyFormBlock.submitButtonText',
+    defaultMessage: 'Submit',
+  },
+});
+
+@injectIntl
+export class AccuracyFormBlock extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    onInputChange: PropTypes.func,
+    setAnalysisMode: PropTypes.func,
+  };
+
+  static defaultProps = {
+    onInputChange: () => {},
+    setAnalysisMode: () => {},
+  };
+
+  constructor(props) {
+    super(props);
+    this.dropDownOptions = [
+      { value: '2', label: '2' },
+      { value: '3', label: '3' },
+      { value: '4', label: '4' },
+      { value: '5', label: '5' },
+      {
+        value: '-1',
+        label: props.intl.formatMessage(messages.numberOfLogLinesAllOption),
+      },
+    ];
+  }
+
+  normalizeValue = (value) => value && `${value}`.replace(/\D+/g, '');
+
+  render() {
+    const { intl } = this.props;
+    return (
+      <React.Fragment>
+        <FormField
+          name="minShouldMatch"
+          containerClassName={cx('accuracy-form-group')}
+          fieldWrapperClassName={cx('accuracy-form-input-wrapper')}
+          label={intl.formatMessage(messages.minimumShouldMatchTitle)}
+          onChange={this.props.onInputChange}
+          normalize={this.normalizeValue}
+          format={String}
+          description={intl.formatMessage(messages.minimumShouldMatchDescription)}
+        >
+          <FieldErrorHint>
+            <InputWithIcon
+              icon="%"
+              iconClass={cx('percent-icon')}
+              maxLength="3"
+              className={cx('mobile-input-disabled')}
+            />
+          </FieldErrorHint>
+        </FormField>
+
+        <FormField
+          name="minDocFreq"
+          containerClassName={cx('accuracy-form-group')}
+          fieldWrapperClassName={cx('accuracy-form-input-wrapper')}
+          label={intl.formatMessage(messages.minimumDocFreqTitle)}
+          onChange={this.props.onInputChange}
+          normalize={this.normalizeValue}
+          format={String}
+          description={intl.formatMessage(messages.minimumDocFreqDescription)}
+        >
+          <FieldErrorHint>
+            <Input maxLength="2" className={cx('mobile-input-disabled')} />
+          </FieldErrorHint>
+        </FormField>
+
+        <FormField
+          name="minTermFreq"
+          containerClassName={cx('accuracy-form-group')}
+          fieldWrapperClassName={cx('accuracy-form-input-wrapper')}
+          label={intl.formatMessage(messages.minimumTermFreqTitle)}
+          onChange={this.props.onInputChange}
+          normalize={this.normalizeValue}
+          format={String}
+          description={intl.formatMessage(messages.minimumTermFreqDescription)}
+        >
+          <FieldErrorHint>
+            <Input maxLength="2" className={cx('mobile-input-disabled')} />
```

Comment:
In all other input-like components we are using "mobileDisabled" prop(which enabled required styles). Why you decide to change the way here?

Reply:
Changed `Input` component. Fixed